docs/unity/part-2.md

```diff
@@ -494,7 +494,7 @@ Replace the implementation of the `GameManager` class with the following.
 public class GameManager : MonoBehaviour
 {
     const string SERVER_URL = "http://127.0.0.1:3000";
-    const string MODULE_NAME = "blackholio";
+    const string DB_NAME = "blackholio";
```

Comment:
```suggestion
    const string MODULE_NAME = "blackholio";
```

I understand the motivation of this one, but we're passing this into a function called `with_module_name`

Reply:
Also any changes here would have to come with a corresponding change to the Blackholio repo.

Reply:
I've made a PR regarding deprecating the with_module_name method and adding a with_database_name method (@gefjon has closed it before because just renaming it would be a breaking change requiring a major semver version bump, so I've reworked it minutes ago). If it's merged, I can go through the docs again and replace the mentions of the deprecated method names there.

Reply:
https://github.com/clockworklabs/SpacetimeDB/pull/2516

Reply:
Let's do that change later and just implement what we can here for now. Let's not let perfect be the enemy of the good in this case.